internal/session/connection_repository.go

```diff
@@ -0,0 +1,69 @@
+package session
+
+import (
+	"context"
+
+	"github.com/hashicorp/boundary/internal/db"
+	"github.com/hashicorp/boundary/internal/errors"
+	"github.com/hashicorp/boundary/internal/kms"
+)
+
+// Repository is the session database repository
```

Comment:
This comment should start with `ConnectionRepository`, and the rest of the comment seems wrong. I think it should be something like:

```go
// ConnectionRepository is the session connection database repository.
```